lms/djangoapps/discussion_api/api.py

```diff
@@ -634,6 +647,7 @@ def update_thread(request, thread_id, update_data):
     # Only save thread object if some of the edited fields are in the thread data, not extra actions
     if set(update_data) - set(actions_form.fields):
         serializer.save()
+        thread_edited.send(sender=None, user=request.user)
```

Comment:
As we discussed, privileged discussion users can edit other people's threads/comments. I think we need to send both who made the change, and who the thread/comment belongs to.

This is the list of things that privileged users can do--

"edit_content", "delete_thread", "openclose_thread", "endorse_comment", "delete_comment", "see_all_cohorts"